ONSdigital/sds
# Sdss 109 automate cloud function deploy

### Motivation and Context
Allows adding release tags to be added to main to trigger the app and cloud function to deploy to preprod.

### What has changed
Added terraform trigger, plus some validation on variables. No tests because it's terraform.

### How to test?
Add a tag to a commit on the sdss-109-automate-cloud-function-deploy (will change to main on approval) branch in the SDS repo, in the correct format (look at other commits for examples). Check the cloud build to see if deployment was successful.
